src/tito/common.py

```diff
@@ -604,6 +605,12 @@ def munge_specfile(spec_file, commit_id, commit_count, fullname=None, tgz_filena
             ))
             continue
 
+        if version_suffix:
+            m = re.match(r'^(\s*Version:\s*)(.+?)\s*$', line)
+            if m:
+                print(f"{m.group(1)}{m.group(2)}{version_suffix}")
+                continue
```

Comment:
I don't know, do we still care about EPEL7? Anyway, it's easy not to break the backward compatibility here.

```
Compiling /builddir/build/BUILDROOT/tito-0.6.26-1.20240927180525922719.pr502.2.g9069fc1.el7_9.x86_64/usr/lib/python2.7/site-packages/tito/common.py ...
  File "/usr/lib/python2.7/site-packages/tito/common.py", line 611
    print(f"{m.group(1)}{m.group(2)}{version_suffix}")
                                                    ^
SyntaxError: invalid syntax
```

Can you please use the `.format` syntax?

Reply:
> I don't know, do we still care about EPEL7? 

EPEL 7 is EOL, so we can't update in Koji -> only chance is to ship in Copr now.  IMVHO good time to forget about Python 2 support...

> Can you please use the .format syntax?

Can do, if you prefer it.

Reply:
We don't even need  `.format()` in this case, used plain `+`.

Reply:
> IMVHO good time to forget about Python 2 support...

I have no problem if we make that decision, drop EPEL7 from tests and so on. I just didn't want to break it by adding one unnecessary f-string :-)